public/gameInfo.php

```diff
@@ -787,7 +788,8 @@ function ResetProgress() {
                     if ($permissions >= Permissions::Developer) {
                         echo "<form action='/request/game/recalculate-points-ratio.php' method='post'>";
                         echo csrf_field();
-                        echo "<input type='hidden' name='game' value='$gameID'>";
+                        echo "<input type='hidden' name='gameId' value='$gameID'>";
+                        echo "<input type='hidden' name='parentGameId' value='$parentGameID'>";
```

Comment:
I don't think it's worth it to pass parent ID all the way through. Just look it up again in the `UpdateGameWeightedPoints`. That will also handle the case where the scheduled task recalculates the game.

https://github.com/RetroAchievements/RAWeb/blob/8360cffc54dfafd1c9500268803034812bfc0d6a/app_legacy/Platform/Commands/UpdateGameWeightedPoints.php#L36

It also eliminates a possible issue where the URL could be called directly with a mismatched `parentGameId`.

Reply:
Good points, I've addressed this in https://github.com/RetroAchievements/RAWeb/pull/1530/commits/c384be10eec370633b6d37d8f2c33714547b872f. The `run()` function now tries to detect if the given game ID has a parent.